### PR TITLE
Add Osrs Profile plugin

### PR DESCRIPTION
**Pull request by mathiasHillmann**
This plugin scrapes the data of the player and sends it to an external server to show your data similar to how RuneScape 3's adventurer's log work.

Data that will be scraped (Not all are tracked at this moment, but will be added as I develop the API and website): Skills, Quests, Slayer monsters log, Collection log, Achievement diaries, Minigames count, Combat tasks, Boss kills.

The website is available at osrsprofile.com

A warning similar to WikiSync would be needed when trying to install the plugin:
image

Profile with the data scraped:
https://osrsprofile.com/player/H1llman
image

If needed I am on the official RuneLite discord by the name of H1llmann.

### CODE REVIEW

**Comment by mathiasHillmann** [comment]
Changing java.net.http to okhttp.

**Comment by mathiasHillmann** [comment]
Changed gson and java.net.http to okhttp.

File: plugins/osrsprofile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/mathiasHillmann/OsrsProfilePlugin.git
+commit=fd45a445e057ace112534f5f4acf5dbc7d297487
+warning=Similar to the official HiScores, this plugin pushes up data about your character that can be viewed by anyone.<br>We don't suggest using this if you are (for example) a PvP-locked hardcore ironman, where broadcasting your<br>quest status could be detrimental to your account by giving away information about what you're currently doing.
```
Comment (Nightfirecat) [inline plugins/osrsprofile:3]:
```suggestion
warning=This plugin submits your player stats and achievements, and IP address to a server not controlled or verified by the RuneLite developers.
```

**Review by Nightfirecat — COMMENTED** [review]
LGTM otherwise

**Pull request merged**